drivers/pwm/Kconfig.sam0_tc

```
@@ -0,0 +1,17 @@
+# Atmel SAM0 TCC as PWM configuration
+
+# Copyright (c) 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_SAM0_TC
+	bool "Atmel SAM0 MCU Family TC PWM Driver"
+	default y
+	depends on DT_HAS_ATMEL_SAM0_TC_PWM_ENABLED
+	help
+	  Enable PWM driver for Atmel SAM0 MCUs using the TC timer/counter.
+
+config PWM_TC_INIT_PRIORITY
+	int "Init Priority"
+	default 50
+	help
+		Device driver initialization priority
```

Nit: two space indent

```suggestion
	  Device driver initialization priority
```

---

Done, thanks